File: hw_06/CPP/cpp_oppwork06/geometry/earth.cpp
#include <iostream>

using namespace std;

class Earth {
    public:
        double G = 6.673e-11;
        double Radius = 6.3781e6;
        double Mass = 5.97e24; 

        void EarthInfo(){
            cout << "Earth: G = " << G << ", Radius = " << Radius << ", Mass = " << Mass << endl;
        }
};  

File: hw_06/CS/cs_oopwork06/cs_oopwork06/geometry/Earth.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace cs_oopwork06
{
    public class Earth
    {
        public double G = 6.673e-11;
        public double Radius = 6.3781e6;

        public double Mass = 5.67e24;

        public void EarthInfo()
        {
            System.Console.WriteLine($"Earth: G = {G}, Radius = {Radius}, Mass = {Mass}");
        }
    }
}